README-PLAYBOOK.md

```diff
@@ -1,164 +1,164 @@
-in automation and maintenance. 
-2. Users can specify, where the certificate goes once received from Venafi. It supports folder locations as well as 
-CAPI store. 
-3. Users can specify, any after installation actions need to be taken. E.g. Restart my apache or restart nginx or any 
-other script one would like to run after the certificate is available in the right location. 
-4. User can mention renewing before, this helps to run the script on daily basis as part of cronjob or automation, the 
-tool will check if certificate exists or the certificate requires renewal before fetching a new certificate based on 
-the values configured. 
-5. Supports many formats of certificate to be stored, PKCS12, JKS, PEM, and CAPI store.
-
-## Some use cases:
-1. A client having 100s of Microsoft servers can have this script as part of the scheduled job.  The tool will run, 
-check if the certificate  present in the CAPI store needs renewal, if yes, it will fetch a new certificate, and install 
-in CAPI. Tool can run any after install script as well. 
-2. Remote machines like ATMs, can use this tool to pull provisioning certificates on startup or regular intervals. Once 
-the certificate is received, it can run any post-processing scripts configured in the tool.
+VCert Playbook functionality solves the "last mile" problem. VCert has historically done a great job fetching certificates, but getting those certificates installed in the right location usually required custom scripting. The Playbook functionality addresses this use case natively. 
+
+> Throughout this article, we use _TLS Protect Datacenter_ to refer to Trust Protection Platform and _TLS Protect Cloud_ to refer to Venafi as as Service.
+
+## Key features of VCert playbook
+
+- You don't have to pass long command line arguments. You can now call VCert with playbook YAML files, which helps with automation and maintenance. 
+- You can specify where the certificate goes, and in which format, once received from Venafi. It supports common keystore formats (PEM, JKS, PKCS#12) in folder locations as well as Windows CAPI store. 
+- You can specify any post-installation actions need to be taken, such as restarting services (apache, nginx) or any other script you would like to run after the certificate is available in the right location. 
+- For certificate renewals, it checks to see if the certificate already exists and if it is due for renewal before requesting a new certificate. This allows you to run a script regularly without renewing certificates that don't need to be.
+- VCert Playbook functionality works with both TLS Protect Cloud and TLS Protect Datacenter.
+
+## Example use cases
+- Renewal parameters can be setup to automatically renew the certificate before it expires. This model assumes that VCert is added to a daily cronjob or is executed on a regular basis by some other automation. Default renewal is 10% of remaining certificate lifetime.
```

Comment:
Same suggestion here and again with some suggested edits:

- **Automated Certificate Renewal**: You can set renewal parameters to have VCert automatically renew certificates before expiration. This approach assumes that VCert is part of a daily cronjob or is executed routinely through other automation methods. By default, renewal occurs at 10% of the remaining certificate lifetime.

- **Effortless API Access Updates**: When using TLS Protect Datacenter, VCert will automatically update API access and refresh tokens within the playbook. This feature ensures continuous operation without manual intervention. It leverages a refresh token to acquire a new access token when needed, an approach that's particularly effective when paired with a long-lasting refresh/grant token and a short-lived access token, such as a 3-year refresh token and a 1-hour access token.